scripts/opposed-wfrp4e.js

```diff
@@ -269,23 +276,27 @@ class OpposedWFRP
   }
 
   // Opposed starting message - manual opposed
-  static createOpposedStartMessage(speaker)
+  static createOpposedStartMessage(speaker, rollMode)
   {
-    ChatMessage.create(
-    {
-      user: game.user._id,
-      hideData: true,
-      flags:{"opposedStartMessage": true},
-      content: `<div><b>${speaker.alias}<b> ${game.i18n.localize("ROLL.OpposedStart")}<div>`
-    }).then(msg => this.startMessage = msg)
+    let content = `<div><b>${speaker.alias}<b> ${game.i18n.localize("ROLL.OpposedStart")}<div>`
+    let chatOptions = WFRP_Utility.chatDataSetup(content, rollMode);
+
+    chatOptions["hideData"] = true;
+    chatOptions["flags"] = {"opposedStartMessage": true};
+
+    ChatMessage.create(chatOptions).then(msg => this.startMessage = msg)
   }
 
-  // Update starting mesasge with result - manual opposed
+  // Update starting message with result - manual opposed
   static updateOpposedMessage(damageConfirmation, msgId)
   {
-    let opposeMessage = game.messages.get(msgId)
+    let opposeMessage = game.messages.get(msgId);
+    let rollMode=opposeMessage.data.rollMode;
+    console.log(`Update - Roll Mode selected : ${rollMode}`);
```

Comment:
I don't think this console.log is needed outside debugging purposes

Reply:
what is your strategy about logging ? only errors but sometimes I see some debugging stuff.

Reply:
Debugging stuff is probably meant to be deleted, I know Edmund has a bunch of odd logging that I haven't deleted yet. The only general log wfrp4e system does is when it plays audio, if I recall